src/modules/update.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, It would be better to **remove the comments to disable linter checks. in all your files** The linter checks are here to help you improve the quality of your code 🙏 Kindly remove all of them as suggested



_Removing comments to disable linter checks in your code is not recommended because comments are an important part of code documentation and can help other developers understand the code better._